packages/eas-json/schema/eas.schema.json

```diff
@@ -398,33 +377,46 @@
-                "default",
+                "auto",
                 "latest",
+                "`sdk-50`",
+                "`sdk-49`",
                 "macos-ventura-13.6-xcode-15.2",
                 "macos-ventura-13.6-xcode-15.1",
-                "macos-ventura-13.6-xcode-15.0",
-                "macos-ventura-13.4-xcode-14.3.1",
-                "macos-ventura-13.3-xcode-14.3",
-                "macos-monterey-12.6-xcode-14.2",
-                "macos-monterey-12.6-xcode-14.1",
-                "macos-monterey-12.6-xcode-14.0"
+                "macos-ventura-13.6-xcode-15.0"
               ],
               "markdownEnumDescriptions": [
-                "`macos-ventura-13.4-xcode-14.3.1`",
-                "`macos-ventura-13.6-xcode-15.2`"
+                "When using this option the build image is selected automatically based on the project configuration, detected Expo SDK and React Native versions.",
+                "The latest iOS image currently available. It is resolved to `macos-ventura-13.6-xcode-15.2`. The `latest` to image mapping will be updated as new images are released.",
+                "The recommended image for SDK 50 builds: `macos-ventura-13.6-xcode-15.2`",
+                "The recommended image for SDK 49 builds: `macos-ventura-13.6-xcode-15.2`"
               ]
             },
             {
               "deprecated": true,
               "enum": [
+                "default",
+                "stable",
                 "macos-monterey-12.4-xcode-13.4",
                 "macos-monterey-12.3-xcode-13.3",
-                "macos-monterey-12.1-xcode-13.2"
+                "macos-monterey-12.1-xcode-13.2",
+                "macos-ventura-13.4-xcode-14.3.1",
+                "macos-ventura-13.3-xcode-14.3",
+                "macos-monterey-12.6-xcode-14.2",
+                "macos-monterey-12.6-xcode-14.1",
+                "macos-monterey-12.6-xcode-14.0"
               ],
               "markdownEnumDescriptions": [
-                "This image is deprecated, please use one of the more recent images instead.",
-                "This image is deprecated, please use one of the more recent images instead.",
-                "This image is deprecated, please use one of the more recent images instead."
+                "This image tag is deprecated, please use other image tags or images directly instead.",
+                "This image tag is deprecated, please use other image tags or images directly instead.",
+                "This image is using Xcode in version lower then 15. Apple's new requirement states that starting on April 29th, 2024, apps uploaded to App Store Connect must be built with Xcode 15 for iOS 17, iPadOS 17, tvOS 17, or watchOS 10. Start using Xcode 15 image today by setting `macos-ventura-13.6-xcode-15.2` or `latest` as your image value. This image will be removed from EAS Build on May 29th, 2024. Learn more: expo.dev/changelog/2024/03-29-eas-build-upcoming-ios-images-updates.",
```

Comment:
Should the URL have https? I _think_ I saw vscode linkifying links in descriptions?